tests/e2e/test_pvc_creation_deletion_scale.py

```diff
@@ -0,0 +1,224 @@
+        Measure PVC deletion time after creation test
+        """
+        number_of_pvc = 1500
+        log.info(f"Start creating {access_mode}-{interface} {number_of_pvc} PVC")
+
+        if interface == constants.CEPHBLOCKPOOL:
+            self.sc_obj = constants.DEFAULT_STORAGECLASS_RBD
+        elif interface == constants.CEPHFS_INTERFACE:
+            self.sc_obj = constants.DEFAULT_STORAGECLASS_CEPHFS
+
+        # Create PVC
+        pvc_objs = helpers.create_multiple_pvcs(
+            sc_name=self.sc_obj,
+            namespace=self.namespace,
+            number_of_pvc=number_of_pvc,
+            size=f"{random.randrange(5, 105, 5)}Gi",
+            access_mode=access_mode
+        )
+
+        # Check for PVC status using threads
+        threads = list()
+        for obj in pvc_objs:
+            process = threading.Thread(
+                target=helpers.wait_for_resource_state,
+                args=(obj, constants.STATUS_BOUND, )
+            )
+            process.start()
+            threads.append(process)
+        for process in threads:
+            process.join()
+
+        # Get pvc_name, require pvc_name to fetch creation time data from log
+        pvc_name_list, pv_name_list = ([] for i in range(2))
+        for pvc_obj in pvc_objs:
+            pvc_obj.reload()
+            pvc_name_list.append(pvc_obj.name)
+            pv_name_list.append(pvc_obj.backed_pv)
+
+        # Get PVC creation time
+        pvc_create_time = helpers.measure_pvc_creation_time_bulk(
+            interface=interface, pvc_name_list=pvc_name_list
+        )
+
+        # TODO: Update below code with google API, to record value in spreadsheet
+        # TODO: For now observing Google API limit to write more than 100 writes
+        csv_obj = csv.writer(open(f"/tmp/{self.sc_obj}-{access_mode}-create.csv", "w"))
+        for k, v in pvc_create_time.items():
+            csv_obj.writerow([k, v])
+        logging.info(
+            f"Create data present in /tmp/{self.sc_obj}-{access_mode}-create.csv file"
+        )
+
+        # Delete PVC
+        for obj in pvc_objs:
+            obj.delete()
+            obj.ocp.wait_for_delete(obj.name)
+
+        # Get PVC deletion time
+        pvc_deletion_time = helpers.measure_pv_deletion_time_bulk(
+            interface=interface, pv_name_list=pv_name_list
+        )
+
+        # Update result to csv file.
+        # TODO: Update below code with google API, to record value in spreadsheet
+        # TODO: For now observing Google API limit to write more than 100 writes
+        csv_obj = csv.writer(open(f"/tmp/{self.sc_obj}-{access_mode}-delete.csv", "w"))
+        for k, v in pvc_deletion_time.items():
+            csv_obj.writerow([k, v])
+        logging.info(
+            f"Delete data present in /tmp/{self.sc_obj}-{access_mode}-delete.csv file"
+        )
+
+    @polarion_id('OCS-1885')
+    @pytest.mark.usefixtures(namespace.__name__)
+    def test_all_4_type_pvc_creation_scale(self, namespace):
```

Comment:
This test case require all 4 PVC type, due to this there are some similar code lines but with different interfaces.